user-service/src/main/java/uk/gov/gchq/palisade/service/user/stream/config/AkkaSystemConfig.java

```diff
@@ -65,7 +65,7 @@ ActorSystem getActorSystem(final PropertiesConfigurer propertiesConfigurer) {
 
     @Bean
     @ConditionalOnMissingBean
-    Materializer getMaterializer(final ActorSystem system) {
+    Materializer getMaterialiser(final ActorSystem system) {
```

Comment:
Still finding a few places where there are serialize and deserialize, e.g. KafkaInitializer class there are places where it needs to be deserialize and others where it needs to be deserialise and in the text for a message in an exception "Failed to deserialize".